### ISSUE TITLE
Documentation of channel cloning/channel RSS feed

### ISSUE THREAD

**Issue by fuglede** [conda/conda#10319]
In the [current Conda channel documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/channels.html#conda-clone-channel-rss-feed), there's a section (added [here](https://github.com/conda/conda/commit/6815ed8772c2cbbe5e2643b945fbef1d97feacb9)) on cloning channels, and on an RSS feed that is updated upon doing so.

From this, I found it hard to figure out exactly what functionality is offered, and how to make use of it:

* The process of "cloning channels" seems undocumented. Is that right? It sounds like something that could be mighty useful: I've cloned custom channels by simply copying all files, but this is error-prone as a single I/O error will leave the channel in an invalid state. If channel cloning offers a more robust approach, that could be great. Or maybe it's something else entirely?
* Along the same lines, does the RSS feed have any meaning outside of channel cloning. For my own custom channels, the only contents I've ever seen are

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE rss PUBLIC "-//Netscape Communications//DTD RSS 0.91//EN" "http://www.rssboard.org/rss-0.91.dtd">
<rss version="0.91">
  <channel>
    <title>{{ channel_name }} updates</title>
    <link>https://anaconda.org</link>
    <description>Updates in the last two weeks</description>
    <language>en</language>
    <copyright>Copyright 2019, Anaconda, Inc.</copyright>
    <pubDate>Mon, 09 Dec 2019 13:27:16 +0000</pubDate>
  </channel>
</rss>
```

**Comment by fuglede** [comment]
Yes, the contents of that page are unchanged since the time this issue was created.

**Comment by Collins-Webdev** [comment]
> Yes, the contents of that page are unchanged since the time this issue was created.

Hello, it's done. You can review the pull requests and proceed with the merge.

**Comment by Collins-Webdev** [comment]
> In the [current Conda channel documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/channels.html#conda-clone-channel-rss-feed), there's a section (added [here](https://github.com/conda/conda/commit/6815ed8772c2cbbe5e2643b945fbef1d97feacb9)) on cloning channels, and on an RSS feed that is updated upon doing so.
> 
> From this, I found it hard to figure out exactly what functionality is offered, and how to make use of it:
> 
> * The process of "cloning channels" seems undocumented. Is that right? It sounds like something that could be mighty useful: I've cloned custom channels by simply copying all files, but this is error-prone as a single I/O error will leave the channel in an invalid state. If channel cloning offers a more robust approach, that could be great. Or maybe it's something else entirely?
> * Along the same lines, does the RSS feed have any meaning outside of channel cloning. For my own custom channels, the only contents I've ever seen are
> 
> ```
> <?xml version="1.0" encoding="UTF-8"?>
> <!DOCTYPE rss PUBLIC "-//Netscape Communications//DTD RSS 0.91//EN" "http://www.rssboard.org/rss-0.91.dtd">
> <rss version="0.91">
>   <channel>
>     <title>{{ channel_name }} updates</title>
>     <link>https://anaconda.org</link>
>     <description>Updates in the last two weeks</description>
>     <language>en</language>
>     <copyright>Copyright 2019, Anaconda, Inc.</copyright>
>     <pubDate>Mon, 09 Dec 2019 13:27:16 +0000</pubDate>
>   </channel>
> </rss>
> ```

Hello, it's done. You can review the pull requests and proceed with the merge.

**Comment by travishathaway** [comment]
Please see my comment here:

https://github.com/conda/conda/pull/13623#issuecomment-1963723128

I think that this section no longer belongs in our documentation as it is specific to [anaconda.org](https://anaconda.org). I propose removing it entirely.

@fuglede, there is no official process for cloning channels. This is not documented because it does not exist. The documentation you are confused about simply refers to the process that [anaconda.org](https://anaconda.org) follows to make packages available to its CDN.

**Comment by travishathaway** [comment]
@fuglede,

I did find out about this project today though:

- https://github.com/conda-incubator/conda-mirror

It might be worth checking out if you are still interested in this sort of thing.
